addOns/ascanrules/src/test/java/org/zaproxy/zap/extension/ascanrules/BufferOverflowScanRuleUnitTest.java

```diff
@@ -40,6 +42,26 @@ protected BufferOverflowScanRule createScanner() {
         return new BufferOverflowScanRule();
     }
 
+    @Test
+    void shouldReturnExpectedMappings() {
+        // Given / When
+        int cwe = rule.getCweId();
+        int wasc = rule.getWascId();
+        Map<String, String> tags = rule.getAlertTags();
+        // Then
+        assertThat(cwe, is(equalTo(120)));
+        assertThat(wasc, is(equalTo(7)));
+        assertThat(tags.size(), is(equalTo(2)));
+        assertThat(tags.containsKey(CommonAlertTag.OWASP_2021_A03.name()), is(equalTo(true)));
```

Comment:
Could you change these to use `getName()`?

Reply:
Done :)